src/components/meshes/Text.js

```diff
@@ -117,8 +119,6 @@ class Text extends MeshComponent {
       });
     });
 
-    super.wait(promise);
```

Comment:
@hirako2000 This was used. Now if build returns a `Promise` - [mesh will automatically wait for it](https://github.com/WhitestormJS/whs.js/blob/dev/src/core/MeshComponent.js#L110).